source/CPluginSystem.cpp

```diff
@@ -25,20 +25,27 @@ void CPluginSystem::LoadPlugins()
 
         for (DWORD i = 0; i < files.count; i++)
         {
+            if (std::find(filenames.begin(), filenames.end(), files.strings[i]) != filenames.end())
+                continue; // file already listed
```

Comment:
how can this condition be true, if there are no two files with the same name in one folder?